bagel/pheno_utils.py

```diff
@@ -35,6 +39,30 @@ def validate_portal_uri(portal: str) -> Optional[str]:
     return portal
 
 
+def load_pheno(input_p: Path) -> pd.DataFrame | None:
+    """Load a .tsv pheno file and do some basic validation."""
+    if input_p.suffix != ".csv":
+        pheno_df: DataFrame = pd.read_csv(
+            input_p, sep="\t", keep_default_na=False, dtype=str
+        )
+
+        if pheno_df.shape[1] > 1:
+            # We only extracted one column, this might mean that we have a sneaky .csv
+            return pheno_df
+        elif len(pheno_df.columns[0].split(",")) == 1:
+            warnings.warn(
+                f"Your phenotypic input file {input_p} has only one column."
+                " Although this is not invalid, such a file is not valid."
```

Comment:
> " Although this is not invalid, such a file is not valid."

I think I get what you're trying to say, but this part of the warning is a bit confusing as it seems contradictory.

How about keeping it simple:
```suggestion
                " Such a file is not a valid Neurobagel phenotypic data file."
```